modules/porous_flow/include/materials/PorousFlowHysteresisOrder.h

```diff
@@ -0,0 +1,61 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "PorousFlowMaterial.h"
+#include "DerivativeMaterialInterface.h"
+#include "PorousFlowConstants.h"
+
+/**
+ * Computes the hysteresis order for use by the hysteretic capillary pressure and
+ * relative-permeability objects
+ */
+class PorousFlowHysteresisOrder : public DerivativeMaterialInterface<PorousFlowMaterial>
+{
+public:
+  static InputParameters validParams();
+
+  PorousFlowHysteresisOrder(const InputParameters & parameters);
+
+  virtual void initQpStatefulProperties() override;
+  virtual void computeQpProperties() override;
+
+protected:
+  /// Liquid phase number
+  const unsigned _liquid_ph_num;
+
+  /// Stringified liquidphase number
```

Comment:
Add space in liquid phase (really minor!)

Reply:
done